packages/better-auth/src/plugins/organization/organization.ts

```diff
@@ -965,7 +965,7 @@ export const organization = <O extends OrganizationOptions>(options?: O) => {
 						userId: ctx.context.session.user.id,
 						organizationId: activeOrganizationId,
 					});
-					if (!member) {
+					if (!member && (options?.requireMembership ?? true)) {
```

Comment:
I think of instead of simply letting anyone add users to any org (possibly themselves which would be a security issue i believe), check if the option is enabled, and if they do, check if the adder's role is set to admin (as per the admin plugin functionality), and allow users to specify other roles which can override this membership check)

Reply:
Cool. That's exactly what I was thinking of for a real solution to this. This change is more of a stub to start the discussion more than anything. So, order of operations will be:
1. Check if the option is enabled. If not fail.
2. Get the admin plugin, if enabled. If not enabled, fail.
3. Check the current user's role against the admin plugin's `adminRoles` option.
4. If the user has one of the configured `adminRoles`, allow the operation to proceed.